packages/attestation-service/src/requestHandlers/attestation.ts

```diff
@@ -221,6 +229,31 @@ class AttestationRequestHandler {
 
     return attestation
   }
+
+  private async verifyPepperIfApplicable(): Promise<void> {
+    if (this.attestationRequest.phoneNumberSignature && this.attestationRequest.salt) {
+      Counters.attestationRequestsProvidedSignature.inc()
```

Comment:
had another thought on this -- we can consider also exposing this info (or a percentageUpgraded, i.e. Counters.attestationRequestsProvidedSignature /  Counters.requests) through the `/status` endpoint + adding to the mainnet_metadata_crawler (and then redeploying that) so that this gets automatically scraped during existing attestation service checks. This way we will have a quick way (+ requiring only minimal data pipeline changes) to get this information

Reply:
What about just having a metric for requests that don't provide a phoneNumberSignature? That way we can make the field non-optional when that metric goes to zero. Seems potentially simpler than configuring a percentage of two metrics?

Reply:
Ooh that's definitely more helpful than the # that did provide requests, though we may still be missing some valuable context; I think your suggestion makes sense @alecps, and maybe we expose the # or requests that don't provide a signature + the total number of requests (Counters.requests) from the `/status` endpoint which is simpler than the percentage, and then we can always compute the percentage in our data pipeline later!

Reply:
changed the metric to track requests that don't provide a signature instead! Will expose to `/status` in a separate PR